lib/bigquery/dataset.js

```diff
@@ -56,6 +56,9 @@ function Dataset(bigQuery, id) {
 /**
  * Create a table given a tableId or configuration object.
  *
+ * See [Tables: insert](https://cloud.google.com/bigquery/docs/reference/v2/tables/insert)
```

Comment:
Wow, that's a microsetting if I've ever seen one. What about JSHint? Do we turn off the rule on JSHint and let JSCS catch breakages?